python/GangaLHCb/Lib/Applications/GaudiExec.py

```diff
@@ -269,6 +272,13 @@ def constructExtraFiles(self, job):
 
         folder_dir = master_job.getInputWorkspace(create=True).getPath()
 
+        extraopts = ' '
```

Comment:
Ok, I'll stick this in a default options file to be added to the job if the user wishes.